### PR TITLE
Rails 6 minor + Sprockets Config Update (assets.rb)

### PR DESCRIPTION
**Pull request by masonhensley**
Rebased against #47 (documentation update) and giving a shot at Travis tests to carry the torch on mrsweater's work in https://github.com/prey/gdpr_rails/pull/45 

Will close #47 if this goes green.

Fixes https://github.com/prey/gdpr_rails/issues/44

## FYI
* Had to update assets.rb & manifest.js to make the latest version of sprockets happy. Was receiving errors like [this](https://stackoverflow.com/questions/34344094/after-gem-update-test-fail-with-asset-was-not-declared-to-be-precompiled-in-pr)

* Updated Apprisals & Documentation for future contributors

## Notes about documentation update:
... Quick documentation update for future contributors based on feedback in #46 & #45 

- Updated contribution documentation
- Added ruby 2.6.4 to travis pipelines matrix for Rails 6

## Update
Ready for review. Thanks for the project 🎉

### CODE REVIEW

File: spec/dummy/app/assets/config/manifest.js [hunk 1]
```diff
@@ -2,4 +2,3 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
-//= link terms_manifest.js
```
Comment (masonhensley) [inline spec/dummy/app/assets/config/manifest.js]: This was failing on Rails 6 tests, could not figure out reasoning behind this file, seems phantom. Also, super weird that the versions of rails prior to 6.0 passed without error.

**Comment by mrsweaters** [comment]
@patriciojofre @michelson @matipacheco Is there anything we need to change to get this merged? I know it can be a pain in the ass maintaining a repo, so we want to as much help as possible here (and hopefully not a hindrance).

**Comment by michelson** [comment]
I think we are ready to merge, just note the change on .update_attributes with .update , is .update compatible with rails 5.1.4 ? 

Le damos @patriciojofre ?

**Comment by mrsweaters** [comment]
In Rails. 5.1.4 `update_attributes` is an alias of `update` so it should be fine.

**Comment by mrsweaters** [comment]
Can we get this merged in?

**Comment by psnider30** [comment]
Any update on when this is getting merged?

**Comment by patriciojofre** [comment]
sorry guys, PR merged! thanks for contribute to gdpr rails!

**Pull request merged**